github/orgs_members.go

```diff
@@ -270,3 +270,33 @@ func (s *OrganizationsService) RemoveOrgMembership(user, org string) (*Response,
 
 	return s.client.Do(req, nil)
 }
+
+//ListPendingOrgInvitations returns a list of contains a role field which refers to the
+//Organization Invitation role and will be one of the following values:
+//direct_member, admin, billing_manager, hiring_manager, or reinstate.
+//If the invitee is not a GitHub member, the login field in the return hash will be null.
```

Comment:
Don't forget to include a space between the text and `//`.

Rationale: https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes